buttons-examples/Web-MWC/web-button-example.md

```diff
@@ -0,0 +1,457 @@
+<!--docs:
+title: "Buttons"
+layout: detail
+section: components
+excerpt: "Web Buttons"
+ide_version: "<cIDE name> <compatible IDE version and build number>"
+material_package_version: "<compatible Material platform package version number>"
+iconId:
+path: /:
```

Comment:
You need to change this to ensure that it's placed into the correct directory by the material.io import script:

path: /catalog/buttons

Reply:
👍